src/NHibernate.Test/NHSpecificTest/NH3985/Mappings.hbm.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<hibernate-mapping xmlns="urn:nhibernate-mapping-2.2" assembly="NHibernate.Test" namespace="NHibernate.Test.NHSpecificTest.NH3985">
+	<class name="Process" table="Processes">
+		<id name="ProcessID" type="Int32">
+			<column name="ProcessID" not-null="true" sql-type="INT" />
+			<generator class="identity" />
+		</id>
+		<property name="Name" type="String">
+			<column name="Name" not-null="true" length="50" sql-type="NVARCHAR(50)" />
```

Comment:
Avoid specifying db specific types. Better not specify any type at all, NHibernate infers them.